dspace-server-webapp/src/test/java/org/dspace/app/rest/WorkspaceItemRestRepositoryIT.java

```diff
@@ -1625,4 +1630,209 @@ public void uploadTest() throws Exception {
         ;
     }
 
+
+
+
+
+
+
+
+
+    @Test
```

Comment:
There's a large number of blank lines above this test which can be removed.

Reply:
This has been updated